app/javascript/Pages/components/Navbar/Navbar.css

```diff
@@ -0,0 +1,169 @@
+.navbar {
+  background: #6600cc;
+  height: 80px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  font-size: 1.2rem;
+  position: sticky;
+  top: 0;
+  z-index: 999;
+}
+
+.navbar-container {
+  display: flex;
+  justify-content: space-between;
+  height: 80px;
+  z-index: 1;
+  width: 100%;
+  max-width: 1300px;
+  margin-right: auto;
+  margin-left: auto;
+  padding-left: 50px;
+}
+
+.navbar-logo {
+  color: #fff;
+  justify-self: start;
+  cursor: pointer;
+  margin-top: 2%;
```

Comment:
**suggestion (non-blocking)**: It's a best practice to use rem, vw/vh or (in last resort) px as units for applying styles. rem is tied to the font size of the browser, so if a user increases the font size all the spacing also increases). vw/vh are relative to the size of the viewport which make them a better alternative to %. The main issue with % is that it can be relative to the window or parent element (depending on css "position"). Also, there is a strange browser bug where using percentages for padding-top /bottom or margin-top /bottom will use the width as base instead of the height 🤷🏻‍♂️